README.rst

```diff
@@ -0,0 +1,318 @@
++-----------------------------------+-----------------------------------+
+| Release on conda-forge            | |Anaconda-Server Badge|           |
++-----------------------------------+-----------------------------------+
+| ubermag/oommf Docker image        | |ubermag-oommf-docker-image|      |
++-----------------------------------+-----------------------------------+
+| License                           | |License Badge|                   |
++-----------------------------------+-----------------------------------+
+
+.. sectnum::
+
+.. contents:: 
+
+OOMMF Git repository
+====================
+
+
+About
+-----
+
+This is a repository of the
+`OOMMF <https://math.nist.gov/oommf/oommf.html>`__ software. It holds
+the OOMMF source files as distributed by
+`NIST <https://www.nist.gov/>`__ available
+`here <https://math.nist.gov/oommf/software.html>`__ in the
+`oommf <oommf>`__ subdirectory of this repository.
+
+The intention is to
+
+1. provide the OOMMF distribution files in a git repository for those who prefer to pull them via git
+
+2. include additional OOMMF extensions, which have not made it into the official distribution from NIST into this git repository (see below).
+
+In addition to the OOMMF source code that is offered from NIST (and the
+OOMMF extension coming from NIST), this repository contains some
+additional user-contributed extensions. Those can be seen from the
+`Makefile-update-oommf <Makefile-update-oommf>`__. See also
+`Extensions <#Extensions>`__ below.
+
+Compilation of OOMMF (for example by running ``make build``) is expected
+to work on Linux and OS X.
+
+Version
+-------
+
+The current version of OOMMF code in this repository is the alpha
+release of OOMMF 2.0 (``20a2 20200608 a2``).
+
+You can check which version you have, by inspecting the content of the
+`oommf-version <oommf-version>`__ file.
+
+The history of all OOMMF versions hosted in this directory is provided
+in `versionlog.txt <versionlog.txt>`__.
+
+Clone and build OOMMF from source
+---------------------------------
+
+If you want to build OOMMF from source, clone the repository:
+
+::
+
+   git clone https://github.com/fangohr/oommf.git
+
+and build it
+
+::
+
+   make build
+
+Smoke test
+----------
+
+As a quick test of the compilation, you can compute standard problems 3
+and 4 as a test of your build by running:
+
+::
+
+   make test-all
+
+Extensions
+----------
+
+A number of OOMMF extensions are included:
+
+We provide Dzyaloshinskii-Moriya interaction extensions for three
+different crystallographic classes:
+
+1. Cnv - interfacial DMI
+   (`repository <https://github.com/joommf/oommf-extension-dmi-cnv>`__),
+2. T(O) - bulk DMI
+   (`repository <https://github.com/joommf/oommf-extension-dmi-t>`__),
+   and
+3. D2d - antiskyrmion DMI
+   (`repository <https://github.com/joommf/oommf-extension-dmi-d2d>`__)
+
+And
+
+-  the OOMMF Magnetoelastic Extension Module
+   (`repository <https://github.com/yuyahagi/oommf-mel>`__) from Yu Yahagi.
```

Comment:
I'm still not quite happy with the list (because MEL is now listed under "DMI extensions". Mabye the following would be an option?
```
A number of OOMMF extensions are include:
- <DMI for different ...>
  1. Cnv
  2. T(O)
  3. D2d
- MEL
 ```

Reply:
Looks like GitHub  would show this with Roman numbers for the second level. Therefore, two levels of bullet points might be better.